docs/source/fates_tech_note.rst

```diff
@@ -2043,61 +2043,233 @@ Stomatal Conductance
 :ref:`Collatz et al. (1991)<Collatzetal1991>` and :ref:`Sellers et al. (1996)<sellers1996>` is
 that they used net photosynthesis (i.e., leaf photosynthesis minus leaf
 respiration) instead of gross photosynthesis. As implemented here,
 stomatal conductance equals the minimum conductance (:math:`b`) when
-gross photosynthesis (:math:`A`) is zero. Leaf stomatal resistance is
+gross photosynthesis (:math:`A`) is zero. Leaf stomatal conductance is
 
 .. math:: \frac{1}{r_{s}} = m_{ft} \frac{A}{c_s}\frac{e_s}{e_i}P_{atm}+b_{ft} \beta_{sw}
 
-where :math:`r_{s}` is leaf stomatal resistance (s m\ :math:`^2`
-:math:`\mu`\ mol\ :math:`^{-1}`), :math:`b_{ft}` is a plant functional
+where :math:`r_{s}` is leaf stomatal resistance (s m\ :math:`^2` leaf area
+:math:`\mu`\ mol :math:`H_{2}O^{-1}`), :math:`b_{ft}` in units of 
+:math:`\mu`\ mol :math:`H_{2}O` m\ :math:`^{-2}` leaf area s\ :math:`^{-1}` is a plant functional
 type dependent parameter equivalent to :math:`g_{0}` in the Ball-Berry
 model literature. This parameter is also scaled by the water stress
 index :math:`\beta_{sw}`. Similarly, :math:`m_{ft}` is the slope of the
-relationship between the assimilation, :math:`c_s` and humidty dependant
-term and the stomatal conductance, and so is equivalent to the
-:math:`g_{1}` term in the stomatal literature. :math:`A` is leaf
-photosynthesis (:math:`\mu`\ mol CO\ :math:`_2` m\ :math:`^{-2}`
-s\ :math:`^{-1}`), :math:`c_s` is the CO\ :math:`_2` partial pressure at
+relationship (i.e. stomatal slope, or the :math:`g_{1}` term in the stomatal literature) 
+between stomatal conductance and the stomatal index, comprised of the leaf assimilation 
+rate, :math:`A` (:math:`\mu`\ mol CO\ :math:`_2` m\ :math:`^{-2}` leaf area s\ :math:`^{-1}`), 
+:math:`c_s` is the CO\ :math:`_2` partial pressure at
 the leaf surface (Pa), :math:`e_s` is the vapor pressure at the leaf
 surface (Pa), :math:`e_i` is the saturation vapor pressure (Pa) inside
-the leaf at the vegetation temperature conductance (:math:`\mu`\ mol
-m\ :math:`^{-2}` s\ :math:`^{-1}`) when :math:`A` = 0 . Typical values
-are :math:`m_{ft}` = 9 for C\ :math:`_3` plants and :math:`m_{ft}` = 4
-for C\ :math:`_4` plants (
-:ref:`Collatz et al. 1991<Collatzetal1991>`, :ref:`Collatz, 1992<Collatzetal1992>`, :ref:`Sellers et al 1996<sellersetal1996>`).
-:ref:`Sellers et al. 1996<sellers1996>` used :math:`b` = 10000 for C\ :math:`_3`
-plants and :math:`b` = 40000 for C\ :math:`_4` plants. Here, :math:`b`
-was chosen to give a maximum stomatal resistance of 20000 s
-m\ :math:`^{-1}`. These terms are nevertheless plant strategy dependent,
-and have been found to vary widely with plant type
-:ref:`Medlyn et al. 2011<Medlynetal2011>`.
-
-Resistance is converted from units of s m\ :math:`^2 \mu`
-mol\ :math:`^{-1}` to s m\ :math:`^{-1}` as: 1 s m\ :math:`^{-1}` =
+the leaf at the vegetation temperature :math:`T_{v}` (K), and :math:`b_{ft}` 
```

Comment:
@Qianyuxuan updated this portion based on our discussion and clarification of the previous confusing, erroneous text

Reply:
@serbinsh Nice work! Looks great to me.